astro

```diff
@@ -197,10 +197,11 @@ typesetgmi() {
 			"> "*) sty="$sty_quote" && line="${line#> }" ;;
 			"=>"*)
 				link=${line#'=>'}
-				echo "${link#' '}" >> "$linksfile"
+				link=${link##' '}
+				echo "$link" >> "$linksfile"
```

Comment:
I think this line can be merged with the previous one.

Reply:
I'll take a look.